dtfs-central-api/src/types/custom-express-request.ts

```diff
@@ -0,0 +1,31 @@
+import { Request } from 'express';
+
+/**
+ * Obtained from the express 'core.ParamsDictionary' type
+ */
+type RequestParams = Record<string, string>;
+
+/**
+ * Obtained from the express 'core.Query' type
+ */
+interface RequestQuery {
+  [key: string]: string | string[] | RequestQuery | RequestQuery[] | undefined;
+}
+
+type CustomExpressRequestOptions = {
+  params?: RequestParams;
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Not a big fan of eslint inline rule disable, anything can be done for an ideal fix?

Reply:
The default type for `import('express').Request.reqBody` and `import('express').Request.resBody` is `any` in the official `@types/express`. This eslint rule was disabled (rather than coming up with a different type) to avoid any unintentional type conflicts between a `CustomExpressRequest` and the default `import('express').Request` type for properties you don't wish to overwrite